src/devices/src/virtio/block/device.rs

```diff
@@ -209,6 +210,7 @@ impl Block {
                             // Stop processing the queue and return this descriptor chain to the
                             // avail ring, for later processing.
                             queue.undo_pop();
+                            METRICS.block.transfer_rate_limiter_throttled.inc();
```

Comment:
What would be de quantitative insight for multiple throttled transfers over a time frame?

Another option would be if we increment with the throttled bytes count. Knowing that metrics are dumped at every 60s, we can make calculations to how many bytes were throttled for that time frame.

Reply:
If we get blocked in `process_queue_event`, we don't pop the first descriptor from the queue so don't know the request data length.